nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -0,0 +1,48 @@
+collision_monitor:
+  ros__parameters:
+    use_sim_time: True
+    base_frame_id: "base_footprint"
+    odom_frame_id: "odom"
+    cmd_vel_in_topic: "cmd_vel_raw"
+    cmd_vel_out_topic: "cmd_vel"
+    transform_tolerance: 0.5
+    data_timeout: 5.0
```

Comment:
`source_timeout` so that as we're writing documentation and we're talking about the "sources" (scan, PC2) deriving from the "source" class, this nomenclature is obvious to them. `data_timeout` is totally fine, I think it can be improved :smile: